py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py

```diff
@@ -239,3 +243,245 @@ def _add_layer(
         input_val = clamp_max_layer.get_output(0)
 
     return input_val
+
+
+def add(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    return convert_binary_elementwise(
+        network, target, source_ir, name, trt.ElementWiseOperation.SUM, lhs_val, rhs_val
+    )
+
+
+def mul(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    return convert_binary_elementwise(
+        network,
+        target,
+        source_ir,
+        name,
+        trt.ElementWiseOperation.PROD,
+        lhs_val,
+        rhs_val,
+    )
+
+
+def max(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    return convert_binary_elementwise(
+        network, target, source_ir, name, trt.ElementWiseOperation.MAX, lhs_val, rhs_val
+    )
+
+
+def min(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    return convert_binary_elementwise(
+        network, target, source_ir, name, trt.ElementWiseOperation.MIN, lhs_val, rhs_val
+    )
+
+
+def sub(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    return convert_binary_elementwise(
+        network, target, source_ir, name, trt.ElementWiseOperation.SUB, lhs_val, rhs_val
+    )
+
+
+def div(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    lhs_val: Union[TRTTensor, int, float],
+    rhs_val: Union[TRTTensor, int, float],
+) -> TRTTensor:
+    if isinstance(lhs_val, TRTTensor):
+        lhs_val = trt_cast_int_to_float(network, name, lhs_val)
+
+    if isinstance(rhs_val, TRTTensor):
+        rhs_val = trt_cast_int_to_float(network, name, rhs_val)
```

Comment:
Consider using `cast_int_int_div_trt_tensor` here instead of these calls. Based on C++ and TorchScript code, it seems TRT will only complain if _both_ inputs are integers, and not if either individual input is an integer. Additionally, consider updating `cast_int_int_div_trt_tensor` to not check for `int8`, which is a special case, as mentioned above.